third_party/embed.go

```diff
@@ -18,5 +18,8 @@ package third_party
 
 import _ "embed"
 
+// TODO: Use this embedded PCI DB as an input to https://github.com/jaypipes/ghw
+// library in order to remove dependency on the PCI DB beign available in the OS.
```

Comment:
```suggestion
// library in order to remove dependency on the PCI DB being available in the OS.
```

What is this blocked on? We should file a tracking issue.

IMHO we should ideally make this work like go's fallback embedded tzdata and prefer the host's *if available* but use the embedded one if not so we don't _rely_ on the host having it.

Then users / distros can provide a more up to date PCI DB via a mount or custom image without patching the core project, but we still get the reliability of not depending on the host.

Reply:
Prior art at: https://pkg.go.dev/time/tzdata

Reply:
ACK, I'll file the follow-up.